log-collector/src/main/scala/io/phdata/pulse/logcollector/PulseKafkaConsumer.scala

```diff
@@ -53,11 +53,12 @@ class PulseKafkaConsumer(solrService: SolrService) extends JsonSupport with Lazy
         val records = consumer.poll(MAX_TIMEOUT)
         for (record <- records.asScala) {
           logger.trace("KAFKA: Consuming " + record.value() + " from topic: " + topic)
-          val logEvent = record
-            .value()
-            .parseJson
-            .convertTo[LogEvent]
-          solrInputStream ! (logEvent.application.get, logEvent)
+          val logEventMap = Util.logEventToFlattenedMap(
```

Comment:
There might be a little too much nesting here, maybe pull out the `record.value...` into its own value?